tests/test_views_rest.py

```diff
@@ -35,7 +35,148 @@
 from flask_security import url_for_security
 from six import BytesIO
 
-from invenio_deposit.api import Deposit
+
+def test_edit_deposit_1(app, db, es, users, location, deposit, write_token):
```

Comment:
I agree with @jirikuncar.  Descriptive symbol names make it easier to read and understand the code snippets quickly without having to scan the wider context.  This is good for reading patches, good for auto-completing symbols, good for jumping around or grepping code, etc. See e.g. the old *"Code Complete"* book that has a very nice chapter about "The power of variable names".

Note that we even have an example against using names like `list2` in our old coding style:

- https://github.com/inveniosoftware/invenio/blob/maint-1.2/modules/webhelp/web/hacking/coding-style.webdoc#L112-L124

Shouldn't be too hard to write descriptive function names too, if you wrote the docstrings alongside?  Even if it may not seem that important in this particular case of test functions, it's good for the overall coherence of the codebase and for the general imprinting and popularisation of good practices...